examples/03_evaluate/als_movielens_diversity_metrics.ipynb

```diff
@@ -36,13 +36,16 @@
     "We evaluate the results obtained with two approaches: using the ALS recommender algorithm vs. a baseline of random recommendations. \n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #10.    als_diversity
</pre><p>this is very interesting, Yan do you know why the diversity varies in comparison between the coocurrence method?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/microsoft/recommenders/pull/1505/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='45'/>

Reply:
the calculation of diversity is based on item-item similarity values. It varies when using different item-item similarity calculation approaches (1) based on item co-occurrence count; and (2) based on item feature vectors.

Reply:
<p>It's interesting that diversity decreased when using the features, for both ALS and random. Maybe you could mention this.</p>

Reply:
added discussion in the notebook